packages/settings/src/Domain/Setting/SettingName.ts

```diff
@@ -1,3 +1,3 @@
-export enum MfaSetting {
+export enum SettingName {
   MfaSecret = 'MFA_SECRET',
 }
```

Comment:
```suggestion
/**
 * Enumerate all Settings with their names as identifiers
 *
 * Extend by adding more enum members
 */
export enum Setting {
  MfaSecret = 'MFA_SECRET',
}

/**
 * Describe Setting payloads (contents) matched with Setting enum keys
 *
 * Extend by adding more properties with keys from Setting enum and value with the corresponding Payload
 */
export type SettingPayload = {
  [Setting.MfaSecret]: { secret: string };
};

```

Reply:
You beat me to the refactor. What do you think about this ^?

Reply:
hmm not sure if we need to define the payload like that - but feel free to add it another PR.

Additionally keep in mind to keep 1 type/class/interface per file